azure-pipelines.yml

```diff
@@ -9,27 +9,17 @@ pr:
 - rel/*
 
 pool:
-  vmImage: windows-2022
+  vmImage: windows-latest
+
+variables:
+  BuildConfiguration: Release
 
 jobs:
 - job: BuildBits
+  displayName: Build and Test solution
   timeoutInMinutes: 60
   steps:
 
-  # Install NuGet
-  - task: NuGetToolInstaller@0
-    displayName: Install NuGet 6.0
-    inputs:
-      versionSpec: 6.0.0
-
-  # Install NerdBank GitVersioning
```

Comment:
While the comment is wrong here, the command updates NBGV to the latest version if the pool image doesn't have it. So why remove it?

Reply:
The pool does have the latest version and I read that the pool image gets updated according to Microsoft's Modern Lifecycle policy and .NET's Current channel.

Just another un-necessary step here.